README.md

```diff
@@ -378,6 +378,9 @@ AAAI Conf. Artificial Intelligence, 2020. [PDF](https://www.aaai.org/Papers/AAAI
 - <a name="Monforte20aicas"></a>Monforte, M., Arriandiaga, A., Glover, A., Bartolozzi, C.,  
 *[Exploiting Event Cameras for Spatio-Temporal Prediction of Fast-Changing Trajectories](https://arxiv.org/pdf/2001.01248)*,  
 IEEE Int. Conf. Artificial Intelligence Circuits and Systems (AICAS), 2020.
+- <a name="Sengupta2021aicas"></a>Sengupta, J. P., Kubendran, R., Neftci, E., Andreou, A. G.,  
+*[High-Speed, Real-Time, Spike-Based Object Tracking and Path Prediction on Google Edge TPU.](https://par.nsf.gov/servlets/purl/10212648)*
```

Comment:
Need two blank spaces at the end of the line on github to force a new line.

Reply:
The two spaces are needed after "TPU", not in the last line after the page numbers

![image](https://user-images.githubusercontent.com/8024432/137157595-82cb736d-e454-4699-afeb-ade7c9a318d8.png)